app/styles/brands/psyarxiv.scss

```diff
@@ -25,5 +25,5 @@
 
 .branded-footer a,
 #view-page .osf-box-lt .fa {
-    color: #ff383f;
+    color: #CF1D35;
```

Comment:
Good to know! Going with a CDN will really simplify some of our code in the future.

Not a blocker for this PR, but speaking in general terms: we probably could still use SASS in provider styles if we wanted to. It should be possible to build SASS separate from ember, as part of the "deploy new styles to production" script.